pkg/operator/ceph/object/user/controller.go

```diff
@@ -281,13 +281,44 @@ func (r *ReconcileObjectStoreUser) createorUpdateCephUser(u *cephv1.CephObjectSt
 		} else {
 			return errors.Wrapf(err, "failed to get details from ceph object user %q", u.Name)
 		}
+	} else if *user.MaxBuckets != *r.userConfig.MaxBuckets {
+		// TODO handle update for user capabilities
```

Comment:
Nit. Let's link to the Ceph issue tracker here so we understand what is blocking Rook currently.

https://tracker.ceph.com/issues/52521